src/main/KeyboardControls.tsx

```diff
@@ -130,30 +119,22 @@ const KeyboardControls: React.FC = () => {
   })
 
   const render = () => {
-    if (keyMap && Object.keys(keyMap).length > 0) {
-
-      const obj: Record<string, Array<KeyMapDisplayOptions>> = {}
-      obj[t("keyboardControls.defaultGroupName")] = []    // For keys without a group
-
-      // Sort by group
-      for (const [, value] of Object.entries(keyMap)) {
-        if (value.group) {
-          if (obj[value.group]) {
-            obj[value.group].push(value)
-          } else {
-            obj[value.group] = [value]
-          }
-        } else {
-          obj[t("keyboardControls.defaultGroupName")].push(value)
-        }
-      }
+    if (KEYMAP && Object.keys(KEYMAP).length > 0) {
 
       const groups: JSX.Element[] = [];
-      for (const key in obj) {
-        if (obj[key].length > 0) {
-          groups.push(<Group name={key as ParseKeys} entries={obj[key]} key={key}/>);
-        }
-      }
+      Object.entries(KEYMAP).forEach(([key, value], index) => {
+        const entries : { [key: string]: string[][] } = {}
+        Object.entries(value).forEach(([, value]) => {
```

Comment:
```suggestion
        Object.values(value).forEach((value) => {
```

Also, `key` and `value` (both, the outer and inner `value`) should probably get more descriptive names. It's currently very generic and without knowing the exact structure of `KEYMAP` one has no idea what these variables contain.